library/linux/Makefile

```diff
@@ -34,9 +37,16 @@ LIBRARIES=$(FLUTTER_ENGINE_LIB_FILE)
 HEADERS=$(shell find include/ -type f -name '*.h')
 SOURCES=$(shell find src/ -type f -name '*.cc')
 
-.PHONY: all
+
+# ================================== Targets ==================================
+
+.PHONY: all clean
+
 all: $(LIBRARY_OUT)
 
+clean:
+	$(RM) -f $(LIBRARY_OUT)
```

Comment:
Isn't `-f` redundant with `$(RM)`?

Reply:
By default, `$(RM)` is equal to `rm`, so this edit is only for readability.

Reply:
The documentation for Make says otherwise
https://www.gnu.org/software/make/manual/html_node/Implicit-Variables.html